src/liquid/components/ld-pagination/ld-pagination.css

```
@@ -0,0 +1,126 @@
+.ld-pagination {
+  --ld-pagination-sm-item-size: 2rem;
+  --ld-pagination-md-item-size: 2.5rem;
+  --ld-pagination-lg-item-size: 3.125rem;
+  --ld-pagination-items-space: var(--ld-sp-4);
+  --ld-pagination-column-size: calc(
+    var(--ld-pagination-item-size) + var(--ld-pagination-items-space)
+  );
+  --ld-pagination-item-size: var(--ld-pagination-md-item-size);
+
+  align-items: center;
+  display: inline-flex;
+  list-style: none;
+  margin: 0;
+  overflow: hidden;
+  /* for the focus outline */
+  padding: var(--ld-sp-1);
+  position: relative;
+
+  &--sm {
+    --ld-pagination-item-size: var(--ld-pagination-sm-item-size);
+  }
+
+  &--lg {
+    --ld-pagination-item-size: var(--ld-pagination-lg-item-size);
+  }
+}
+
+.ld-pagination__arrow:not(:last-child),
+.ld-pagination__sticky {
+  margin-right: var(--ld-pagination-items-space);
+}
+
+.ld-pagination__sticky,
+.ld-pagination__item {
+  width: var(--ld-pagination-item-size);
+
+  ld-button::part(button) {
+    padding: 0;
+  }
+}
+
+.ld-pagination__dots,
+.ld-pagination__item {
+  transition: opacity 100ms ease-in-out;
+}
+
+.ld-pagination__dots {
+  color: var(--ld-thm-primary);
+  opacity: 0;
+  pointer-events: none;
+  position: relative;
+
+  &--visible {
+    opacity: 1;
+  }
+
+  &--end {
+    right: var(--ld-pagination-column-size);
+  }
+
+  span {
+    align-items: center;
+    justify-content: center;
+    display: flex;
+    height: 100%;
+    position: absolute;
+    white-space: nowrap;
+    width: var(--ld-pagination-item-size);
+  }
+}
+
+.ld-pagination__slide-wrapper {
+  height: var(--ld-pagination-item-size);
+  position: relative;
+  width: calc(
+    var(--ld-pagination-slider-cols) * var(--ld-pagination-column-size)
+  );
+}
+
+.ld-pagination__items {
+  display: inline-flex;
+  list-style: none;
+  padding: 0;
+  pointer-events: none;
+  position: absolute;
+  transform: translateX(
+    calc(
+      var(--ld-pagination-slide-index) * var(--ld-pagination-column-size) * -1
+    )
+  );
+  transition: transform 100ms ease-in-out;
+}
+
+.ld-pagination__item {
+  left: calc(var(--ld-pagination-item-pos) * var(--ld-pagination-column-size));
+  opacity: 0;
+  position: absolute;
+
+  &--visible {
+    opacity: 1;
+    pointer-events: auto;
+  }
+
+  &--selected ld-button {
+    --ld-button-bg-color: transparent;
+
+    &::part(button) {
+      color: var(--thm-primary-active);
+      font-weight: 900;
+    }
+  }
+}
+
+.ld-pagination__marker {
+  background: var(--ld-thm-primary-alpha-lowest);
+  border-radius: var(--ld-br-m);
+  height: var(--ld-pagination-item-size);
+  position: absolute;
+  width: var(--ld-pagination-item-size);
+  z-index: -1;
+  transform: translateX(
+    calc(var(--ld-pagination-column-size) * var(--ld-pagination-selected-index))
+  );
+  transition: transform 100ms ease-in-out;
```

```suggestion
  transition: transform 100ms ease;
```

`ease` is like `ease-in-out`, except it starts slightly faster than it ends. In my opinion it feels smoother.
https://developer.mozilla.org/en-US/docs/Web/CSS/easing-function